tests/message/message_renames.proto

```diff
@@ -7,30 +7,44 @@ import "patch/go.proto";
 option go_package = "github.com/alta/protopatch/tests/message";
 
 message Francis {
-	option (go.message).name = 'Frank';
+  option (go.message).name = 'Frank';
 }
 
 message OriginalOneofMessage {
-	option (go.message).name = 'RenamedOneofMessage';
-	oneof contents {
-		int32 id = 1;
-		string name = 2;
-	};
+  option (go.message).name = 'RenamedOneofMessage';
+  oneof contents {
+    int32 id = 1;
+    string name = 2;
+  };
 }
 
 message OriginalOuterMessage {
-	option (go.message).name = 'RenamedOuterMessage';
-	message InnerMessage {}
-	InnerMessage inner = 1;
+  option (go.message).name = 'RenamedOuterMessage';
+  message InnerMessage {}
+  InnerMessage inner = 1;
 }
 
 message OuterMessageWithRenamedInnerMessage {
-	message InnerMessage {
-		option (go.message).name = 'RenamedInnerMessage';
-	}
-	InnerMessage inner = 1;
+  message InnerMessage {
+    option (go.message).name = 'RenamedInnerMessage';
+  }
+  InnerMessage inner = 1;
 }
 
 message MessageWithRenamedField {
-	int32 id = 1 [(go.field).name = 'ID'];
+  int32 id = 1 [(go.field).name = 'ID'];
+}
+
+message MessageWithEmbeddedFields {
```

Comment:
I’d break apart these into discrete messages for each behavior you want to validate. Also, it might be confusing to have a type called `message`.